src/components/Empty/Empty.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+import { Empty } from './';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Empty', () => {
+    /*
+     * Functionality Tests
+     */
+    test('empty renders', () => {
+        const wrapper = mount(<Empty description="Test" />);
+        expect(wrapper.containsMatchingElement(<Empty />)).toEqual(true);
+    });
```

Comment:
we need to begin writing more meaningful tests to avoid growing technical debt

Reply:
Agreed, stubbing this one in to unblock Table. There are really only 2 this component needs, may add them before merging.

Reply:
Ack